### PR TITLE
203 rule of thumb public pensions

### PR DESCRIPTION
**Pull request by ZhuZhuThanou**
### Description
The rule of thumb learning content page for non-finalized EN version

List of proposed changes:
- added the rule of thumb public pension page under learning
- added content data for EN/FR placeholder
- linked from the main learning page 

### What to test for/How to test
npm i
npm run dev
open the following links in a browser
http://localhost:3000/fr/learn/rule-of-thumb-public-pensions
http://localhost:3000/en/learn/rule-of-thumb-public-pensions
### Additional Notes

### CODE REVIEW

File: frontend/src/pages/learn/rule-of-thumb-public-pensions.tsx [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+import { GetServerSideProps } from 'next'
+import { Trans, useTranslation } from 'next-i18next'
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+import { NextSeo } from 'next-seo'
+
+import {
+  LearnPageLayout
+} from '../../components/LearnPageLayout'
+
+export interface ImportantCardProps extends PropsWithChildren { }
```
Comment (gregory-j-baker) [inline frontend/src/pages/learn/rule-of-thumb-public-pensions.tsx:13]: Bit of a nitpick, but this doesn't have to be exported if it isn't used outside of this module.

File: frontend/src/pages/learn/rule-of-thumb-public-pensions.tsx [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+const ImportantCard: FC<ImportantCardProps> = ({ children }) => (
+  <p className="rounded-lg bg-[#ffe8a3]/[.3] p-5">{children}</p>
+)
+
+export interface DisclaimerCardProps extends PropsWithChildren { }
```
Comment (gregory-j-baker) [inline frontend/src/pages/learn/rule-of-thumb-public-pensions.tsx:18]: Bit of a nitpick, but this doesn't have to be exported if it isn't used outside of this module.

**Review by gregory-j-baker — APPROVED** [review]
Approved w/ comments.

File: frontend/src/pages/learn/rule-of-thumb-public-pensions.tsx (hunk 1 above)
Comment (ZhuZhuThanou) [reply frontend/src/pages/learn/rule-of-thumb-public-pensions.tsx:13]: removed the export keyword

File: frontend/src/pages/learn/rule-of-thumb-public-pensions.tsx (hunk 2 above)
Comment (ZhuZhuThanou) [reply frontend/src/pages/learn/rule-of-thumb-public-pensions.tsx:18]: removed the export keyword

**Comment by ZhuZhuThanou** [comment]
I also removed the export key word from ImportantCardProps / DisclaimerCardProps retirement-income-sources and from-work-to-retirement to keep things consistent.

**Pull request merged**